server/cmwell-ws/app/controllers/Application.scala

```diff
@@ -3001,6 +3001,35 @@ callback=< [URL] >
     }
   }
 
+  def handleStpControl(agent: String) = Action.async(parse.raw) { implicit req =>
+
+    isAdminEvenNonProd(req) match {
+      case true => {
+
+        val headers = req.cookies.get("X-CM-WELL-TOKEN2") match {
+          case Some(cookie) => Seq(("X-CM-WELL-TOKEN", cookie.value))
+          case _ => Nil
+        }
+
+        val flag = req.getQueryString("enabled").flatMap(asBoolean).getOrElse(true)
```

Comment:
I don't like the fact we are making a _in post from Application.scala, where we can simply invoke a `crudServiceFS` method... @nicklester, how complicated will it be to do that instead?

Reply:
Have done this - please can you take a look (FWIW - I completely agree 👍 )

Reply:
@bryaakov made another push..